platform/ios/MGLMapView.mm

```diff
@@ -272,7 +272,10 @@ - (BOOL)commonInit
     _mbglFileSource = new mbgl::DefaultFileSource([MGLFileCache obtainSharedCacheWithObject:self]);
 
     // Start paused on the IB canvas
-    _mbglMap = new mbgl::Map(*_mbglView, *_mbglFileSource, mbgl::MapMode::Continuous, _isTargetingInterfaceBuilder);
+    _mbglMap = new mbgl::Map(*_mbglView, *_mbglFileSource, mbgl::MapMode::Continuous);
+    if (_isTargetingInterfaceBuilder) {
+        _mbglMap->pause();
```

Comment:
OK, it does seem to work in Interface Builder designables, and `MGLMapView`s instantiated from storyboards seem to work fine.